installer/PowerToysBootstrapper/bootstrapper/bootstrapper.cpp

```diff
@@ -190,7 +213,15 @@ int bootstrapper(HINSTANCE hInstance)
             LPWSTR* argList = CommandLineToArgvW(GetCommandLineW(), &nCmdArgs);
             for (int i = 1; i < nCmdArgs; ++i)
             {
-                params += argList[i];
+                if (((std::wstring)argList[i]).find(' ') != std::string::npos)
+                {
+                    params += L"\"" + (std::wstring)argList[i] + L"\"";
+                }
```

Comment:
Let's make it a bit more clear:
```suggestion
                if (std::wstring_view{argList[i]}.find(L' ') != std::wstring_view::npos)
                {
                    params += L'"';
                    params += argList[i];
                    params += L'"';
                }
```